README.md

```diff
@@ -1,16 +1,16 @@
-# Semmle QL
+# CodeQL
 
-This open source repository contains the standard QL libraries and queries that power [LGTM](https://lgtm.com), and the other products that [Semmle](https://semmle.com) makes available to its customers worldwide.
+This open source repository contains the standard CodeQL libraries and queries that power [LGTM](https://lgtm.com), and the other products that [Semmle](https://semmle.com) makes available to its customers worldwide.
 
-## How do I learn QL and run queries?
+## How do I learn CodeQL and run queries?
 
-There is [extensive documentation](https://help.semmle.com/QL/learn-ql/) on getting started with writing QL.
+There is [extensive documentation](https://help.semmle.com/QL/learn-ql/) on getting started with writing CodeQL.
 You can use the [interactive query console](https://lgtm.com/help/lgtm/using-query-console) on LGTM.com or the [QL for Eclipse](https://lgtm.com/help/lgtm/running-queries-ide) plugin to try out your queries on any open-source project that's currently being analyzed.
```

Comment:
I see that we use "open source" in the first sentence, and "open-source" here. I don't know which one is correct, but we should be internally consistent!

Reply:
A search of the existing QL documentation reveals 14 "open source" and 1 "open-source", so it looks as if this should lose the hyphen. (For the longer term, it looks as if the MS Guide suggests that we should hyphenate this when it's an adjective that precedes a noun, but it's probably better to make that change wholesale.